src/scss/components/_footer.scss

```diff
@@ -1,74 +1,247 @@
+$color-border-gray:#dcdacb;
+
+$padding-top--16: 16px;
+$padding-bottom--16: 16px;
+$padding-left--32: 32px;
+$padding-right--32: 32px;
+
 footer {
-    background: #5d8fa1;
-    padding: 20px 0px 30px 0px;
+    .footer-banner {
+        .footer-section-banner {
+            padding-top: 16px;
+            padding-bottom: 16px;
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            background: $color-white;
+            border-top: 1px solid $color-border-gray;
+            padding-left: 32px;
+            padding-right: 32px;
 
-    @media screen and (max-width: 750px) {
-        padding: 25px 0px 20px 0px;
+            .udlite-heading-lg {
+                font-weight: 700;
+                line-height: 1.2;
+                letter-spacing: -.2px;
+                font-size: 19px;
+            }
+
+            .udlite-text-md {
+                font-weight: 400;
+                line-height: 1.4;
+                font-size: 16px;
+                padding: 8px 0;
+            }
+
+            .banner-button-container {
+                flex-shrink: 0;
+                padding-left: 8px;
+
+                a {
```

Comment:
Seem like btn-link, could you please provide class name for this tag and use it to styles.
example: <a class="btn-link btn-primary"></a>